### ISSUE TITLE
Import error with Next.js in production mode

### ISSUE THREAD

**Issue by andtos90** [configcat/js-ssr-sdk#33]
### Describe the bug

I have a Next application (v12.0.5) and I want to get the feature flags from Configcat on both server side and client side. 

In development everything works as expected but the production version trigger this exception run-time:

```
TypeError: Cannot read properties of undefined (reading 'createClientWithAutoPoll')
    at createClient (/Users/redacted/redacted/IL.PE.PEP.Frontend/node_modules/.pnpm/configcat-js-ssr@3.6.0/node_modules/configcat-js-ssr/lib/index.js:12:17)
    at eval (webpack-internal:///./lib/configcat/index.ts:12:83)
    at Object../lib/configcat/index.ts (/Users/redacted/redacted/IL.PE.PEP.Frontend/platforms/teacher/.next/server/pages/_app.js:198:1)
    at __webpack_require__ (/Users/redacted/redacted/IL.PE.PEP.Frontend/platforms/teacher/.next/server/webpack-runtime.js:33:42)
    at eval (webpack-internal:///./pages/_app.page.tsx:12:72)
    at Function.__webpack_require__.a (/Users/redacted/redacted/IL.PE.PEP.Frontend/platforms/teacher/.next/server/webpack-runtime.js:106:13)
    at eval (webpack-internal:///./pages/_app.page.tsx:1:21)
    at Object../pages/_app.page.tsx (/Users/redacted/redacted/IL.PE.PEP.Frontend/platforms/teacher/.next/server/pages/_app.js:319:1)
    at __webpack_require__ (/Users/redacted/redacted/IL.PE.PEP.Frontend/platforms/teacher/.next/server/webpack-runtime.js:33:42)
    at __webpack_exec__ (/Users/redacted/redacted/IL.PE.PEP.Frontend/platforms/teacher/.next/server/pages/_app.js:1021:39)
```

### To reproduce

Create a Next.js app and use the `getAllValues` below on both client and server side:

```
import * as configcat from "configcat-js-ssr";

const configCatClient = configcat.createClient(YOUR_KEY_HERE);

export const getAllValues = async () => {
  const settingValues = await configCatClient.getAllValuesAsync(userObject);
  return settingValues;
};
```

### Expected behavior

It should work in development and production mode without errors.

### Findings

The issue is related to the `createClient` function that internally uses a reference to `this` that is undefined. I managed to avoid the error using `createClientWithAutoPoll` instead of `createClient`.

I really suggest changing how this works internally or at least updating the readme to avoid an unpleasant surprise in production.

**Comment by laliconfigcat** [comment]
Hello @andtos90 ,

Thanks for reporting the issue, I am fixing it.

**Comment by laliconfigcat** [comment]
I just published v3.7.2. I hope it will solve your problem.

**Comment by andtos90** [comment]
@laliconfigcat it worked, thanks for the quick solution!
